frontend/src/pages/HistoryPage.vue

```diff
@@ -286,10 +92,10 @@ export default {
                     'type' : 'Feature',
                     'geometry' : {
                         'type' : 'Point',
-                        'coordinates' : [place.longitude, place.latitude]
+                        'coordinates' : [place.place.longitude, place.place.latitude]
```

Comment:
Duplicating key like `place.place` is not good, use different name for root object like `checkin`.